Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs

```diff
@@ -120,6 +120,11 @@ public class ManagerConnection
         /// </summary>
         public event EventHandler<AgentLogoffEvent> AgentLogoff;
         /// <summary>
+        /// An AgentRingNoAnswer is triggered when an agent was rang and did not answer.<br/>
+        /// To enable AgentRingNoAnswer you have to set eventwhencalled = yes in queues.conf.<br/>

```

Comment:
A `<br/>` at the end of the text make no sense.

Reply:
those were there already

Reply:
oh i guess they weren't, maybe they copied over with the text? either way, removed them